Meadowlark-js/backends/meadowlark-local-mongodb-opensearch/src/repository/Create.ts

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+import { Collection, Db, MongoClient } from 'mongodb';
+import { DocumentInfo, Security, ValidationOptions, PutResult } from '@edfi/meadowlark-core';
+import { Entity } from '../model/Entity';
+
+/**
+ * No connection pooling and even worse, a new connection per query
+ *
+ */
+async function naiveGetEntities(): Promise<Collection<Entity>> {
+  const client = new MongoClient('mongodb://mongo1:27017,mongo2:27018,mongo3:27019');
```

Comment:
Can we put that URI into config?

And what happens if someone only has one node running (let's say they have just `mongodb://singleNode.whatever.com` in their config). Transaction won't work ... at all? Or just partial? That is, do we actually get a failure with our code, or is it just that we don't get the full benefits?